Mathlib/Algebra/BigOperators/Intervals.lean

```diff
@@ -99,6 +99,11 @@ theorem prod_Ico_add [OrderedCancelAddCommMonoid α] [ExistsAddOfLE α] [Locally
   convert prod_Ico_add' f a b c using 2
   rw [add_comm]
 
+@[to_additive]
+theorem prod_Ico_eq_add_sub (f : ℕ → M) (a b c : ℕ) :
+    (∏ x ∈ Ico a b, f x) = ∏ x ∈ Ico (a + c) (b + c), f (x - c) := by
```

Comment:
I think it's worth generalising slightly (e.g., to cover `α := ℤ`). I'd also flip the two sides and add `simp`. So maybe something like:
```suggestion
@[to_additive, simp]
theorem prod_Ico_add_right_sub_eq [OrderedCancelAddCommMonoid α] [ExistsAddOfLE α] [LocallyFiniteOrder α]
    [Sub α] [OrderedSub α] (a b c : α) :
    ∏ x ∈ Ico (a + c) (b + c), f (x - c) = ∏ x ∈ Ico a b, f x := by
```